bff/src/Bff/Endpoints/DefaultBackchannelLogoutService.cs

```diff
@@ -13,13 +15,12 @@
 using Microsoft.IdentityModel.JsonWebTokens;
 using Microsoft.IdentityModel.Tokens;
 
-// ReSharper disable once CheckNamespace
-namespace Duende.Bff;
+namespace Duende.Bff.Endpoints;
 
 /// <summary>
 /// Default back-channel logout notification service implementation
 /// </summary>
-public class DefaultBackchannelLogoutService(
+internal class DefaultBackchannelLogoutService(
```

Comment:
Validation of logout tokens is an extensibility point today, so we should consider adding a replacement for that. Users might want custom token validation parameters, or might want to customize even further to allow logout tokens to be JWEs.

Reply:
created issue for this